grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -73,9 +72,6 @@ export class RootBaseStore {
   @observable
   onCallApiUrl: string;
 
-  @observable
-  navMenuItem: NavMenuItem;
```

Comment:
this was the only spot where `NavMenuItem` was referenced. However, `navMenuItem` is not referenced in either codebase, I think it is safe to remove?